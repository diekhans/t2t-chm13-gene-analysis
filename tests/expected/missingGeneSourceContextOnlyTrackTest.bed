chr1	89294	213579267	cntx0	0	+	89294	213579267	250,250,210				
chr11	127203	139612	cntx1	0	+	127203	139612	250,250,210				
chr17	64098	41276697	cntx2	0	+	64098	41276697	250,250,210				
chr17	41255383	83227721	cntx3	0	+	41255383	83227721	250,250,210				
chr5	181306501	181324685	cntx4	0	+	181306501	181324685	250,250,210				
chr6	170612621	170738536	cntx5	0	+	170612621	170738536	250,250,210				
chr8	72600	79775	cntx6	0	+	72600	79775	250,250,210				
chrY	25378299	25394719	cntx7	0	+	25378299	25394719	250,250,210				
