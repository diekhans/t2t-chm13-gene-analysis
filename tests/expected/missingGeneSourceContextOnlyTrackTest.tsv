id	name	location	genes
cntx0	cntx0	chr1:89294-213579267	ENSG00000238009.6,ENSG00000239906.1,ENSG00000241860.7,ENSG00000235146.2,ENSG00000229905.1,ENSG00000237491.10,ENSG00000177757.2,ENSG00000228794.10,ENSG00000225880.5
cntx1	cntx1	chr11:127203-139612	ENSG00000230724.9
cntx2	cntx2	chr17:64098-41276697	ENSG00000280279.1
cntx3	cntx3	chr17:41255383-83227721	ENSG00000262898.3,ENSG00000262094.1
cntx4	cntx4	chr5:181306501-181324685	ENSG00000238035.8
cntx5	cntx5	chr6:170612621-170738536	ENSG00000287411.1,ENSG00000230423.3,ENSG00000234986.1,ENSG00000225532.1
cntx6	cntx6	chr8:72600-79775	ENSG00000253896.3
cntx7	cntx7	chrY:25378299-25394719	ENSG00000228786.5
